### ISSUE TITLE
scrollTo does NOT work on named element

### ISSUE THREAD

**Issue by autoaim800** [appium/java-client#342]
## Description
Method `public MobileElement scrollTo(String text)` from https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/ios/IOSElement.java does NOT work on named element (once the element has been assigned with accessibilityIdentifier).

THIS IS A RE-OPEN OF https://github.com/appium/java-client/issues/250, previously closed by @TikhomirovSergey 
THIS ISSUE EXISTS ON iOS ONLY.

### Preliminary analysis:
Attribute of accessibilityIdentifier will assign static value to the 'name', e.g. my-cell-title, hence 'name' can not be used as identifier of changing text, see current implementation below
```return (IOSElement) findElementByIosUIAutomation(".scrollToElementWithPredicate(\"name CONTAINS '" + text + "'\")"); ```

### Likely solution
```return (IOSElement) findElementByIosUIAutomation(".scrollToElementWithPredicate(\"value CONTAINS '" + text + "'\")");```

## Environment
* Issue exists on java client till now - git revision ad9f757  on Dec 6, 2015

## Details
If an element is assigned with accessibilityIdentify=myCellTitle, then the name is always equal to 'myCellTitle', but its 'value' is always changing according to its assignment, hence 'value' should be used as a query parameter instead of 'name'.

## Code To Reproduce Issue [ Good To Have ]
This issue is found during code-static-review, not applicable with code-to-reproduce.

## Exception stacktraces
This issue is found during code-static-review, not applicable with exception.

## Link to Appium logs
This issue is found during code-static-review, not applicable with appium log.

**Comment by TikhomirovSergey** [comment]
@autoaim800 
Now I'm seeing the great description and I'll start the investigation

**Comment by SrinivasanTarget** [comment]
appium/appium#2942

**Comment by SrinivasanTarget** [comment]
appium/appium#4633

**Comment by SrinivasanTarget** [comment]
Android Gestures API: http://developer.android.com/training/gestures/scroll.html

**Comment by TikhomirovSergey** [comment]
I'm closing it according to #385
